index.md

```diff
@@ -26,6 +28,45 @@ like circuit breakers or request rates.
 Kiali also includes Jaeger Tracing
 to provide distributed tracing out of the box.
 
+## Getting started on OpenShift
+
+Kiali is still in development. Snapshots releases are pushed on Dockerhub from the CI pipeline.
+
+To deploy Kiali to your Istio-enabled OpenShift cluster you can run the following. Kiali currently requires Istio version 0.7.1.
+
+```
+export NAMESPACE=istio-system
+ 
+oc create -f https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml -n ${NAMESPACE}
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
+   IMAGE_NAME=kiali/kiali \
+   IMAGE_VERSION=latest \
+   NAMESPACE=${NAMESPACE} \
+   VERBOSE_MODE=4 envsubst | oc create -n ${NAMESPACE} -f -
+
+```
+
+If you don't have `envsubst` installed, you can get it via the Gnu `gettext` package.
+
+Once the above has completed and the Docker image has been pulled from Dockerhub, go to the OpenShift console, select the istio-system project and determine the base-URL of Kiali
+
+![OpenShift console](/assets/img/os-console.png)
+
+In this case it is `http://kiali-istio-system.192.168.64.13.nip.io`.
```

Comment:
you should tell them, "But in your network environment the IP address might be different than what you see here (192.168.64.13)".